tools/module_analyzer.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
+
+# Copyright 2017-present Samsung Electronics Co., Ltd. and other contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
```

Comment:
The `print_function` is used at multiple places like line 48-49. The import of this function will guarantee that the `print` calls are the same in python 2.x and 3.x. (see: https://docs.python.org/2/library/__future__.html and https://www.python.org/dev/peps/pep-3105)